game.libretro.chailove/addon.xml.in

```diff
@@ -20,7 +20,7 @@
 		<summary lang="en">ChaiLove v0.14.0</summary>
 		<description>ChaiLove is a 2D game framework.
 
-Inspired by the Love2D API, ChaiLove allows the creation of 2D games using the ChaiScript scripting language. See https://github.com/RobLoach/ChaiLove for a list of games that are available in ChaiLove.
+Inspired by the Love2D API, ChaiLove allows the creation of 2D games using the ChaiScript scripting language. See https://github.com/libretro/libretro-chailove for a list of games that are available in ChaiLove.
```

Comment:
Only because I've written hundreds of descriptions for emulators and controller add-ons, I find two medium sentences or three short sentences look best in the add-on dialog:

ChaiLove is a 2D game framework. Inspired by the Love2D API, ChaiLove allows the creation of 2D games using the ChaiScript scripting language.

See https://github.com/libretro/libretro-chailove for ChaiLove games.
 
Supported files: .chai, .chailove